### PR TITLE
Added Missile Warfare To Community Addons

### PR DESCRIPTION
**Pull request by koiboi-dev**
I've created a plugin for slimefun and added it to the community projects of the addons list.

### CODE REVIEW

File: pages/Addons.md [hunk 1]
```diff
@@ -72,6 +72,7 @@ DankTech2 | Sefiraat | <details><summary>expand</summary>DankTech is a plugin th
...
 SfChunkInfo | FN_FAL113 | <details><summary>expand</summary>Scan the amount of slimefun blocks in a chunk using a single command.</details> | [Readme](https://github.com/FN-FAL113/SfChunkInfo/blob/main/README.md) | [GitHub](https://github.com/FN-FAL113/SfChunkInfo) | [Builds](https://thebusybiscuit.github.io/builds/FN-FAL113/SfChunkInfo/main/)
 VillagerUtil | Apeiros-46B | <details><summary>expand</summary>VillagerUtil adds a few tools that allow for easy transportation of villagers, curing zombified villagers, cycling trades and more.</details> | [Readme](https://github.com/Apeiros-46B/VillagerUtil/blob/main/README.md) | [GitHub](https://github.com/Apeiros-46B/VillagerUtil) | [Builds](https://thebusybiscuit.github.io/builds/Apeiros-46B/VillagerUtil/main)
+MissileWarfare | Kaiyan2006 | <details><summary>expand</summary>Missile Warfare adds a bunch of missiles to slime fun that can be fired at others!</details> | [GitHub](https://github.com/koiboi-dev/MissileWarfare) | N/A | [Releases] (https://github.com/koiboi-dev/MissileWarfare/releases)
```
Comment (Sefiraat) [inline pages/Addons.md:75]:
```suggestion
MissileWarfare | Kaiyan2006 | <details><summary>expand</summary>Missile Warfare adds a bunch of missiles to Slimefun that can be fired at others!</details> | [GitHub](https://github.com/koiboi-dev/MissileWarfare) | N/A | [Releases] (https://github.com/koiboi-dev/MissileWarfare/releases)
```

Comment (Sefiraat) [inline pages/Addons.md:75]: Is the addon called MissileWarfare or Missile Warfare - you call it both in here - one or the other I think :)

Comment (svr333) [reply pages/Addons.md:75]: how did I let that past :notlikethis:

**Comment by Phoenix-Starlight** [comment]
Huh, some random conflict... must be because I merged! Well, I leave it to you to either sync it up or fix it. @koiboi-dev

**Comment by Phoenix-Starlight** [comment]
Ok I resolved the conflicts for you. Please look over the files again and change and make any corrections as needed. (And reviews to be resolved). I will look at this again once you're happy with it.

**Comment by koiboi-dev** [comment]
Ok I think I'm happy with how the wiki implementation looks

**Review by TheBusyBiscuit — CHANGES_REQUESTED** [review]

**Review by koiboi-dev — COMMENTED** [review]
Oh I didn't see that, sorry for not noticing stuff like this

**Comment by koiboi-dev** [comment]
It should be all good now, sorry for making this such a hassle

**Review by Sefiraat — APPROVED** [review]
LGTM 👍

**Review by Phoenix-Starlight — APPROVED** [review]
Congrats on making it through!

**Review by svr333 — APPROVED** [review]

**Pull request merged**